app/src/main/java/com/onboarding/calculatorkotlin/util/Constants.kt

```diff
@@ -0,0 +1,11 @@
+package com.onboarding.calculatorkotlin.util
+
+object Constants {
+
+    const val EMPTY_STRING = ""
+    const val MINUS = "-"
+    const val ANY = 0
```

Comment:
this is not a representative name for that value, pls change it

Reply:
Solved